frontend/src/features/properties/map/research/ResearchTabsContainer.tsx

```diff
@@ -0,0 +1,54 @@
+import {
+  ResearchTabNames,
+  ResearchTabs,
+  TabResearchView,
+} from 'features/mapSideBar/tabs/ResearchTabs';
+import ActivityListView from 'features/research/activities/list/ActivityListView';
+import { Api_ResearchFile } from 'models/api/ResearchFile';
+import React, { useState } from 'react';
+
+import ResearchSummaryView from './detail/ResearchSummaryView';
+
+export interface IResearchTabsContainerProps {
+  researchFile: Api_ResearchFile;
+  setEditMode: (isEditing: boolean) => void;
+}
+
+/**
+ * container responsible for logic related to map sidebar display. Synchronizes the state of the parcel detail forms with the corresponding query parameters (push/pull).
+ */
+export const ResearchTabsContainer: React.FunctionComponent<IResearchTabsContainerProps> = ({
+  researchFile,
+  setEditMode,
+}) => {
+  const tabViews: TabResearchView[] = [];
+
+  tabViews.push({
+    content: <ResearchSummaryView researchFile={researchFile} setEditMode={setEditMode} />,
+    key: ResearchTabNames.researchDetails,
+    name: 'Research Details',
+  });
+
+  tabViews.push({
+    content: (
+      <ActivityListView researchFileId={researchFile.id} hideFilters={false}></ActivityListView>
```

Comment:
if this view is to have other types of files then ```researchFileId``` should be renamed

Reply:
renamed to fileId